docs/docs/installation/MIGRATION.md

```diff
@@ -0,0 +1,18 @@
+# Migration
+
+If you already have an existing Data Center product deployment, you can migrate it to a Kubernetes cluster using the Data Center Helm charts. 
+
+You will need to migrate your database and your shared home, then all you need to do is to follow the [Installation guide](INSTALLATION.md), using your migrated resources instead of provisioning new ones.
+
+* Migrating your database - To migrate your database, you should point the Helm charts to the existing database or to a migrated version of the database. Do this by updating the `database` stanza in the `values.yaml` file as explained in the [Configure database step in the installation guide](INSTALLATION.md#3-configure-database).
+* Migrating your shared home - Application nodes should have access to a shared directory in the same path. Examples of what the shared file system stores include plugins, shared caches, repositories, attachments, and avatars. Configure your shared home by updating the `sharedHome` stanza in the `values.yaml` file as explained in the [Configure persistent storage step in the installation guide](INSTALLATION.md#5-configure-persistent-storage).
+
+Helpful links:
+[Helm charts](https://helm.sh/){.external}
+
+* [Atlassian Data Center migration plan](https://confluence.atlassian.com/enterprise/atlassian-data-center-migration-plan-935363952.html){.external} - gives some guidance on overall process, organizational preparedness, estimated time frames, and app compatibility. 
+* [Atlassian Data Center migration checklist](https://confluence.atlassian.com/enterprise/atlassian-data-center-migration-checklist-935383667.html){.external} - also provides useful tests and checks to perform throughout the moving process.
+* [Migrating to another database](https://confluence.atlassian.com/doc/migrating-to-another-database-148867.html){.external} - describes how to migrate your Confluence data from your existing database to another database.
```

Comment:
`Migrating Confluence to another database`

Reply:
Should we then mention other products as well - e.g. https://confluence.atlassian.com/adminjiraserver/switching-databases-938846867.html ?

Reply:
Yeah, thanks. I added this link to Jira, but couldn't find a BB page for this.